### ISSUE TITLE
CAM: Unable to add a toolbit to a job and to create a new one

### ISSUE THREAD

**Issue by jkvfc** [FreeCAD/FreeCAD#14765]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Toolbits can only be added to a job by double clicking but not by pressing the push button 'Add to Job'. It's always grayed out.

![ToolSelectorPushButtonAddToJobGrayedOut](https://github.com/FreeCAD/FreeCAD/assets/136894212/d6d2b163-5a8d-4c44-acdc-f49ee53c93c8)

If you try to create a new toolbit you get the following error message after entering the toolbit file name. 

![ErrorMessageToolBitCreation](https://github.com/FreeCAD/FreeCAD/assets/136894212/9ffef2c4-52de-4be4-9bbc-36ba87c31016)

The toolbit file is created properly but there is no entry in the library file.

 

### Full version info

```shell
OS: Ubuntu 22.04.4 LTS (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.37735 (Git)
Build type: Unknown
Branch: main
Hash: 2a6a3649aff0eb5c6a54e67899ed5cc5c6332265
Python 3.10.12, Qt 5.15.3, Coin 4.0.0, Vtk 7.1.1, OCC 7.6.3
Locale: English/United States (en_US)
Installed mods: 
  * FeedsAndSpeeds 0.5.0
  * MyScripts
```

### Subproject(s) affected?

CAM/Path

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
Is this a duplicate of https://github.com/FreeCAD/FreeCAD/issues/11833 ?

**Comment by jkvfc** [comment]
@maxwxyz,
I think the two problems described here are not addressed in #11833. Both are new in version 0.22.0dev not present in version 0.21.1.
